guides/common/modules/con_virtual-machine-subscriptions-overview.adoc

```diff
@@ -0,0 +1,23 @@
+[id="virtual-machine-subscriptions-overview"]
+= Virt-who and virtual machine subscriptions
+
+Virtual machines (VMs) require host-based subscriptions instead of physical subscriptions.
```

Comment:
question: Virt-who is related to RHEL, isn't it?

```suggestion
Virtual machines (VMs) require host-based RHEL subscriptions instead of physical subscriptions.
```

Does this also relate to any other RH content?

Reply:
Looks like only RHEL. Will fix.